tests/end-to-end/regression/5487.phpt

```diff
@@ -13,6 +13,7 @@ PHPUnit %s by Sebastian Bergmann and contributors.
 
 Runtime: %s
 Configuration: %s
+Random Seed: %s
```

Comment:
A change in the configuration file for PHPUnit's own test suite must not have an impact on an end-to-end test such as this one. Adapting the expectation just hides the real problem.

Reply:
This should be solved by b6d22871a1d16e9fa125d457b6fc6baba68d592b.

Reply:
great catch, thanks.